packages/server/src/adapters/node-http/types.ts

```diff
@@ -31,11 +31,32 @@ export type NodeHTTPCreateContextOption<
       createContext: NodeHTTPCreateContextFn<TRouter, TRequest, TResponse>;
     };
 
+export interface NodeMiddlewareLike<
```

Comment:
Drop `export`, add `@internal`, call it connect middleware (I think that's the original name)

```suggestion
/** @internal **/
interface ConnectMiddleware<
```